File: 2-semestre/uri-online-judge/c/1080.c
#include <stdio.h>
 
int main() {
    int i;
    int valores[100];
    int maior = 0;
    int posicao = 0;
    int aux = 0;
    
    for (i = 0; i < 100; i++) {
        scanf("%d", &valores[i]);
        
        if (valores[i] > aux) {
            maior = valores[i];
            posicao = i+1;
            aux = valores[i];
        }
    }
    
    printf("%d\n", maior);
    printf("%d\n", posicao);
 
    return 0;
}

File: 2-semestre/uri-online-judge/java/1080.java
import java.io.IOException;
import java.util.Scanner;

public class Main {
    public static void main(String[] args) throws IOException {
        int[] valores = new int[100];
        int maior = 0;
        int posicao = 0;
        int aux = 0;
        
        Scanner entrada = new Scanner(System.in);
        
        for (int i = 0; i < 100; i++) {
            valores[i] = entrada.nextInt();

            if (valores[i] > aux) {
                maior = valores[i];
                posicao = i+1;
                aux = valores[i];
            }
        }
        
        System.out.printf("%d\n", maior);
        System.out.printf("%d\n", posicao);
    }
}
